tickets.md

```diff
@@ -33,13 +33,15 @@ The venue has amazing grounds to relax and wander in and boast a gym, sauna, ste
 
 ### What are the COVID mitigations?
 
-COVID-19 rates across Europe are still high. We want to keep attendees safe, while at the same time allowing for as much familiarity and community spirit as is possible. For this reason our policy at the conference will be:
+COVID-19 still causes illness and long-term harm across Europe. We want to keep attendees safe, while at the same time allowing for as much familiarity and community spirit as is possible. For this reason our policy at the conference will be:
 
 <ul>
 <li>masks will be provided and should be worn in all indoor conference spaces (except while eating/drinking)</li>
 <li>lateral flow tests will be provided for each day (conference organisers should be notified if the test is positive)</li>
 </ul>
 
+(For the next edition (2026), we hope to sort out active air filtering so we can make masks recommended instead of mandatory.)
+
```

Comment:
I'm not sure if that's worth mentioning or if we omit this altogether? I mean it as "bear with us one more year".

Reply:
(we discussed this during a call: decided to leave the mention here)